requirements.txt

```diff
@@ -15,4 +15,6 @@ pysimdjson
 cloudpathlib
 apache_beam
 datasets
-huggingface_hub
\ No newline at end of file
+huggingface_hub
+loguru
```

Comment:
do we want a seperate requirements of preprocessing or a unified one? i dont have a super strong prefrence here

Reply:
I'm of the opinion that a combined requirements is best here - `huggingface_hub` seems like something we want anyway.